src/website/app/pages/ComponentDoc/index.js

```diff
@@ -39,7 +39,6 @@ type Props = {
   description?: React$Node,
   doc: Object,
   examples?: Array<any>,
-  hidePropDoc?: boolean,
```

Comment:
I don't remember what this was for. I think @brentertz might have put it in

Reply:
As there are no longer any references to it, the need for this prop was likely removed during all of the recent website work.